starkit/example_c.py

```diff
@@ -0,0 +1,21 @@
+def primes(n):
```

Comment:
These are example files, please remove them

Reply:
Yes. But if these are removed, the example tests will fail. So shall I remove the example tests as well?